src/main/java/com/skynest/models/ApiConstants.java

```diff
@@ -5,6 +5,11 @@
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public final class ApiConstants {
-    public static final String BASE_URL = "http://13.94.241.83:8080";
     public static final String REGISTER_ENDPOINT = "/public/register";
+    public static final String LOGIN_ENDPOINT = "/public/login";
+    public static final String ME_ENDPOINT = "/users/me";
+    public static final String GET_ALL_USERS_ENDPOINT = "/users";
```

Comment:
No need to have multiple constants for a single string value. Just have a single one, perhaps called "USERS_PATH"